### PR TITLE
Bump bindgen to 0.66.1

### PR DESCRIPTION
**Pull request by tgonzalezorlandoarm**
Signed-off-by: Tomás González <tomasagustin.gonzalezorlando@arm.com>

### CODE REVIEW

File: cryptoki-sys/Cargo.toml [hunk 1]
```diff
@@ -12,7 +12,7 @@ repository = "https://github.com/parallaxsecond/rust-cryptoki"
 documentation = "https://docs.rs/crate/cryptoki-sys"
 
 [build-dependencies]
-bindgen = { version = "0.63.0", optional = true }
+bindgen = { version = "0.66.0", optional = true }
```
Comment (gowthamsk-arm) [inline cryptoki-sys/Cargo.toml:15]: Can you set it to `0.66.1`?
https://github.com/parallaxsecond/rust-psa-crypto/blob/main/psa-crypto-sys/Cargo.toml#L15

Comment (tgonzalezorlandoarm) [reply cryptoki-sys/Cargo.toml:15]: I can, but the way cargo works is that it will pick up the latest anyways. The bindings were generated with 0.66.1

Comment (tgonzalezorlandoarm) [reply cryptoki-sys/Cargo.toml:15]: Will do the change in a sec

Comment (tgonzalezorlandoarm) [reply cryptoki-sys/Cargo.toml]: Done

**Review by ionut-arm — COMMENTED** [review]
Thanks, good to see this brought up to date.

**Review by ionut-arm — APPROVED** [review]

**Review by gowthamsk-arm — APPROVED** [review]

**Pull request merged**